packages/fiori/src/i18n/messagebundle.properties

```diff
@@ -132,6 +132,24 @@ SHELLBAR_OVERFLOW = More
 #XACT: ARIA announcement for the cancel button
 SHELLBAR_CANCEL = Cancel
 
+#XACT: ARIA announcement for aria-label attribute of Wizard navigation header
+WIZARD_NAV_ARIA_LABEL = Wizard Progress Bar
+
+#XACT: ARIA announcement for aria-label attribute of Wizard list
+WIZARD_LIST_ARIA_LABEL = Wizard Steps
+
+#XACT: ARIA announcement for aria-label attribute of Wizard ActionSheet actions
```

Comment:
not part of the scope.. this is an accessible name for an action sheet, nothing to do with this dialog